JhipsterBlazor/JhipsterBlazor.csproj

```diff
@@ -72,6 +72,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Build" Version="3.2.0-rc1.20223.4" PrivateAssets="all" />
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="3.2.0-rc1.20223.4" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="2.1.76" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Why use Newtonsoft.Json ? if possible i prefer System.Net.Http.Json who is already present to avoid looking for too much dll